dev/io.openliberty.checkpoint/resources/io/openliberty/checkpoint/resources/CheckpointMessages.nlsprops

```diff
@@ -24,7 +24,15 @@
 CHECKPOINT_DUMP_INITIATED_CWWKC0451=CWWKC0451I: A server checkpoint was requested. The server will stopped when the checkpoint completes.
 CHECKPOINT_DUMP_INITIATED_CWWKC0451.explanation=A checkpoint was requested. The server is halted and its current state is written to disk.
 CHECKPOINT_DUMP_INITIATED_CWWKC0451.useraction=Check the knowledge center for details of the checkpoint/restore operations.
-#
+
 CHECKPOINT_RESTORE_CWWKC0452I=CWWKC0452I: The liberty server is resuming execution from a checkpoint.
 CHECKPOINT_RESTORE_CWWKC0452I.explanation=A restore operation was run on a liberty server checkpoint image.
 CHECKPOINT_RESTORE_CWWKC0452I.useraction=Check the knowledge center for details of the checkpoint/restore operations.
+
+CHECKPOINT_FAILED_CWWKC0453E=CWWKC0453E: The server checkpoint request failed with the following message: {0}
+CHECKPOINT_FAILED_CWWKC0453E.explanation=A checkpoint request for the server has failed.  The server has shutdown as a result.
+CHECKPOINT_FAILED_CWWKC0453E.useraction=Check the server logs/checkpoint directory for additional logs to determine why the checkpoint failed.
+
+RESTORE_FAILED_CWWKC0454E=CWWKC0454E: The server failed to restore from a checkpoint with the following message: {0}
+RESTORE_FAILED_CWWKC0454E.explanation=An attempt to restore the server from a checkpoint failed.  The server has shutdown as a result.
```

Comment:
suggest
RESTORE_FAILED_CWWKC0454E.explanation=An attempt to restore the server from a checkpoint failed.  As a result, the server is not started.

Reply:
I've updated the message.